### ISSUE TITLE
Unable to Load Metrics from DAX Studio Connecting to SSAS 2014

### ISSUE THREAD

**Issue by sqlcan** [DaxStudio/DaxStudio#532]
Trying to export VPAX file from DAX Studio (2.14.1) fails while connect to SSAS 2014 (12.0.6372).

1. DAX Studio
2. Connect to SSAS
3. Change model to database.
4. Go to Advance
5. Go to Export Metrics.

Get error:
Error exporting metrics: Query (1, 10) This is not a valid DAX expression.

**Comment by dgosbell** [comment]
Thanks for logging this. Can you hold down the Shift key while starting DAX Studio to enable the [diagnostic logging](https://daxstudio.org/documentation/troubleshooting/logging-settings/) then reproduce this issue and attach the log file to this issue?

**Comment by sqlcan** [comment]
[DaxStudio-20210128.log](https://github.com/DaxStudio/DaxStudio/files/5889999/DaxStudio-20210128.log)

**Comment by dgosbell** [comment]
At the point where it's failing it is just attempting to do a query like the following to get accurate row counts for each table:
```
EVALUATE 
   UNION( 
        ROW( "Table", "<table1>", "Cardinality", COUNTROWS( '<table1>' ) ),
        ROW( "Table", "<table2>", "Cardinality", COUNTROWS( '<table2>' ) ),
        ROW( "Table", "<table3>", "Cardinality", COUNTROWS( '<table3>' ) )
)
```

All of the functions used in this query should be supported back to SSAS 2012, but I'm struggling to recall if they changed the UNION function at some point from just supporting 2 parameters to supporting a variable amount of parameters.

Can you connect to your database and try running the following query to test if the UNION function works on your server with more than 2 table parameters:

```
EVALUATE UNION(ROW("RowNumber", 1), ROW("RowNumber", 2), ROW("RowNumber", 3) )
```

If this works can you run the query below against your model and share the results? As if the above works the issue might be something in one of your table names (although we do have code that should be escaping any special characters in the table names).
```
SELECT DISTINCT [TABLE_NAME] 
FROM $SYSTEM.DBSCHEMA_TABLES 
WHERE [TABLE_TYPE] = 'SYSTEM TABLE'
```

**Comment by sqlcan** [comment]
EVALUATE UNION(ROW("RowNumber", 1), ROW("RowNumber", 2), ROW("RowNumber", 3) )
What I got in return was:
Executing the query ...
Query (1, 8) The syntax for 'EVALUATE' is incorrect.
Run complete

**Comment by dgosbell** [comment]
Hmm so maybe my theory is correct and the older engines only support two tables in UNION. However I would have expected an error about an incorrect number of parameters to the UNION function... so that is a bit strange.

Sorry to keep following up, but I don't have an SSAS 2014 instance I can test on. Does the following work for you where there are just 2 rows?

EVALUATE UNION( ROW("RowNumber", 1), ROW("RowNumber", 2) )

**Comment by sqlcan** [comment]
We actually only tried it with two and it failed.  Interesting bit of information I discovered is the model we are trying to assess is set to 11.

**Comment by SharmaAntriksh** [comment]
@dgosbell  UNION is appearing in orange color, looks like it isn't supported by SSAS 2014
![2](https://user-images.githubusercontent.com/73054674/106382393-5e8ea780-63e5-11eb-960a-d72fca35f0f7.png)

**Comment by marcosqlbi** [comment]
I implemented a logic to disable statistics for SSAS 2012/2014 because the loading would be too slow if we use another approach. This way you will see the statistics in View Metrics, but without additional information like missing keys for invalid relationships and accurate cardinality for columns (the number reported could be larger in some cases).
This setting will appear in 2.14.2 version.

**Comment by SharmaAntriksh** [comment]
Thank you! one question regarding UNION and CROSSFILTER, I saw on DaxGuide that they are marked as available for versions 2012 and 2016 but not for 2014, did something happen back then that they have excluded it from 2014 version?

**Comment by marcosqlbi** [comment]
Probably I fixed an issue in a configuration table - there were a few builds of Excel 2016 that were mapped to SSAS 2012.
The system will take 1-2 days to update the indexes and the published functions, then it should be reflected in the online version of the DAX Guide.
Thanks for reporting it!

**Comment by SharmaAntriksh** [comment]
Thanks a lot Marco!!!

**Comment by dgosbell** [comment]
Closing as this was fixed in 2.14.2
